composer.lock

```diff
@@ -4,26 +4,26 @@
         "Read more about it at https://getcomposer.org/doc/01-basic-usage.md#installing-dependencies",
         "This file is @generated automatically"
     ],
-    "content-hash": "a8fa2a1c68958367fd1e4786958003fb",
+    "content-hash": "336bfc592360903bb751eaf8d23816fa",
     "packages": [
         {
             "name": "guzzlehttp/guzzle",
-            "version": "7.5.0",
+            "version": "7.6.1",
```

Comment:
had to `composer update` this package and `google/cloud-storage ` for composer validation to pass. 🤷‍♂️